mxnet/inference/docker/1.6.0/py3/mkl_blas/Dockerfile.cpu

```diff
@@ -0,0 +1,130 @@
+FROM ubuntu:16.04
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="3"
```

Comment:
This is the same as an existing Dockerfile right? Please make this change in place of that Dockerfile - we should not have two with the same major version to avoid conflicting releases

Reply:
Updated